papis/downloaders/usenix.py

```diff
@@ -0,0 +1,107 @@
+import re
+from typing import Optional
+from urllib.parse import urlparse
+
+import papis.downloaders.base
+
+
+class Downloader(papis.downloaders.Downloader):
+
+    def __init__(self, url: str):
+        super().__init__(
+            url,
+            "usenix",
+            expected_document_extension="pdf",
+        )
+        self._raw_data: Optional[str] = None
```

Comment:
We still support python 3.5 (at least until the next release), so this needs to be
```suggestion
        self._raw_data = None  # type: Optional[str]
```
and similar in other places.